pubsub/topics.rb

```diff
@@ -347,13 +432,16 @@ def publish_messages_async_with_concurrency_control project_id:, topic_name:
         set_topic_policy                                <project_id> <topic_name>                     Set topic policies
         test_topic_permissions                          <project_id> <topic_name>                     Test topic permissions
         create_pull_subscription                        <project_id> <topic_name> <subscription_name> Create a pull subscription
+        create_ordered_pull_subscription                <project_id> <topic_name> <subscription_name> Create a pull subscription with ordering enabled
         create_push_subscription                        <project_id> <topic_name> <subscription_name> Create a push subscription
         publish_message                                 <project_id> <topic_name>                     Publish message
         publish_messages_with_batch_settings            <project_id> <topic_name>                     Publish messages in batch
         publish_message_async                           <project_id> <topic_name>                     Publish messages asynchronously
         publish_message_async_with_custom_attributes    <project_id> <topic_name>                     Publish messages asynchronously with custom attributes
         publish_messages_async_with_batch_settings      <project_id> <topic_name>                     Publish messages asynchronously in batch
         publish_messages_async_with_concurrency_control <project_id> <topic_name>                     Publish messages asynchronously with concurrency control
+        publish_ordered_messages                         <project_id> <topic_name>                     Publish messages asynchronously with ordering keys
```

Comment:
An extra space in this line needs to be removed to fix column alignment.

Reply:
Done.